src/pages/singleLineMap/index.tsx

```diff
@@ -79,64 +79,66 @@ const SingleLineMapPage = () => {
 
   return (
     <PageContainer className="map-container">
-      <Typography className="page-title" variant="h4">
-        {t('singleline_map_page_title')}
-        <InfoYoutubeModal
-          label={t('open_video_about_this_page')}
-          title={t('time_based_map_page_description')}
-          videoUrl="https://www.youtube-nocookie.com/embed/bXg50_j_hTA?si=inyvqDylStvgNRA6&amp;start=93"
-        />
-      </Typography>
-      <Grid container spacing={2} sx={{ maxWidth: INPUT_SIZE }}>
-        <Grid container spacing={2} size={{ xs: 12 }}>
-          {/* choose date*/}
-          <Grid size={{ sm: 4, xs: 12 }}>
-            <DateSelector time={dayjs(timestamp)} onChange={handleTimestampChange} />
+      <div className="position-stiky-header">
+        <Typography className="page-title" variant="h4">
```

Comment:
why not using the same element?
```suggestion
        <Typography className="page-title position-sticky-header" variant="h4">
```